Sources/JWT/JWT.swift

```diff
@@ -19,6 +20,9 @@ extension Application {
 
         let application: Application
 
+        public let appleJWKS: JWKSCache
```

Comment:
I think `app.jwt.apple.keys` would read better here. This would also give us a place to configure the apple identifier:
```swift
app.jwt.apple.identifier = "..."
```

Reply:
I'm willing, but it's not entirely clear that these are JWKS then if you just call it apple.

Reply:
`app.jwt.apple.jwks` or `app.jwt.apple.keys` seems pretty clear to me.

Reply:
This also gives us a place to expose the signers future (currently locked away in the verify method) which could be nice:

```swift
let signers = try app.jwt.apple.signers.wait()
print(signers) // JWTSigners
```

Reply:
To what end?